dll/shellext/netshell/lanconnectui.cpp

```diff
@@ -26,15 +26,16 @@ CNetConnectionPropertyUi::~CNetConnectionPropertyUi()
 }
 
 VOID
-AddItemToListView(HWND hDlgCtrl, PNET_ITEM pItem, LPWSTR szName, BOOL bChecked)
+AddItemToListView(HWND hDlgCtrl, PNET_ITEM pItem, LPWSTR szName, BOOL bChecked, UINT Image)
 {
     LVITEMW lvItem;
 
-    ZeroMemory(&lvItem, sizeof(lvItem));
```

Comment:
why is the zeromemory not useful anymore? are all the other non-set fields zeroed out?

Reply:
Only mask, iItem and iSubItem are required, the rest are specified by mask.

Reply:
ok but then we're passing a structure to the listview function with some uninitialized stuff.
Since we're in C++ here, you could do: `LVITEMW lvItem = {0};`.

Reply:
`{0}` is just going to `memset` anyway here.

My code is correct, you have to specify which item you are setting/getting and then MSDN says this about the mask:

> Set of flags that specify **which members of this structure contain data to be set** or which members are being requested.